docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md

```diff
@@ -8,14 +9,79 @@ redirect_from:
+
+- **Decide how email sending should be handled.** If you want to send emails using Spryker, decide whether you want to use the Mail Service that Spryker PaaS offers or attach your own Mail Server to your project. In case you want to use ours, validate your DNS name and lift sending restrictions. Let us know the email address you want to send from, and we can help you with the validation. If you have already delegated the NS records, we can set it ourselves.
+- **Delegate DNS (Optional)**. To find out how to delegate a DNS name, see [Setting up a custom SSL certificate](https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/setting-up-a-custom-ssl-certificate.html).
+
+{% info_block infoBox "Note" %}
+	
+The DNS delegation should be done after clarifying your migration plan. Once these records are set on your side, the DNS configuration is done by Spryker. This lets us configure and verify everything without the need for action on your side. However, if you have multiple DNS records set for your domain name and plan to set more DNS records later, Spryker needs to manage those for you in the future. Also, to delegate your DNS, send us your DNS name's current zone file to set all the records on our side.
+	
+{% endinfo_block %}
+
+{% info_block warningBox "Warning" %}
+
+If you do not delegate DNS to Spryker, decide how to deal with customers reaching your shop using the root domain (in our example, this is spryker.com). Spryker cannot provide you with an IP to point the DNS name to because Spryker PaaS works with DNS names for its endpoints exclusively.  You also cannot set a `CNAME` for a root domain. This means that you either find a way to redirect your visitors via another endpoint or delegate your DNS to us to resolve this issue.
+
+{% endinfo_block %}
+
+## Three weeks before go-live
+
+- **Verify that your deploy.yml file is set up correctly**. Verify that your project works and works with the final endpoints. You can set both testing and final endpoints in your deploy.yml file. Your developers can now mock a "live" operation of the project with its final endpoints by adjusting their local host entries and test your shop thoroughly.
+- **TLS certificates to be used are provisioned**. In case of delegation DNS to Spryker, TLS certificates for your endpoints should be created automatically. If you want us to create a TLS certificate for your endpoints but do not want to delegate your DNS, we can provide you with the verification records. If you have not received them already, contact us using the [Support Portal](https://support.spryker.com). If you do not delegate your DNS, make sure to provide us with the TLS certificates as detailed in the [Setting up a custom SSL certificate](https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/setting-up-a-custom-ssl-certificate.html#next-step) instruction so we can configure the certificates for you.
```

Comment:
```suggestion
- **TLS certificates to be used are provisioned**. In case you delegate DNS to Spryker, TLS certificates for your endpoints should be created automatically. If you want us to create a TLS certificate for your endpoints but do not want to delegate your DNS, we can provide you with the verification records. You can request them via the [Support Portal](https://support.spryker.com). If you do not delegate your DNS, make sure to provide us with the TLS certificates as described in [Setting up a custom SSL certificate](https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/setting-up-a-custom-ssl-certificate.html#next-step) so we can configure the certificates for you.
```